assets/app/widgets/suse-lazy-spinner/suse-lazy-spinner.directive.spec.js

```diff
@@ -0,0 +1,125 @@
+/*global bard $httpBackend assert $rootScope $compile $timeout */
+describe('SUSE Lazy Spinner Directive', function () {
+
+    var directiveElem,
+        scope;
+
+    beforeEach(function () {
+        bard.appModule('crowbarApp');
+
+        bard.inject('$rootScope', '$compile', '$httpBackend', '$timeout');
+
+        //Mock requests that are expected to be made
+        $httpBackend.expectGET('app/features/upgrade/i18n/en.json').respond({});
+        $httpBackend.flush();
+
+        scope = $rootScope.$new();
+    });
+
+    function compileDirective(tpl) {
+        var elem = angular.element(tpl);
+        var compiledElem = $compile(elem)(scope);
+        scope.$digest();
+
+        return compiledElem;
+    }
+
+    // Verify no unexpected http call has been made
+    bard.verifyNoOutstandingHttpRequests();
+
+    describe('directive created with default parameters', function () {
+        beforeEach(function () {
+            directiveElem = compileDirective('<suse-lazy-spinner active="testActive">');
```

Comment:
Shouldn't this be instead:
`'<suse-lazy-spinner active="testActive"></suse-lazy-spinner>'`